changelogs/fragments/rds_global_cluster_info.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
No changelog fragment is needed for new modules. You just need to add skip-changelog label.